index.html

```diff
@@ -4,23 +4,23 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Github Session 2019</title>
+    <title>Open Hackday 2019</title>
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
     <div class="container">
         <img src="images/GitHub-Mark.png" alt="Avatar" class="image">
         <div class="overlay">
             <div class="text">
-                <h1 class="title white">
+                <h1 class="title yellow">
```

Comment:
Thank you for your contribution. If you are using `yellow` class here, you have to change it in the CSS file as well.